mkt/developers/templates/developers/payments/premium.html

```diff
@@ -239,14 +239,20 @@ <h2 id="paid-tab-header">
                           {% endfor %}
                         </tbody>
                       </table>
+                      <p class="hint note rest-of-world">
+                        {% trans %}
+                          Because "Rest of World" is selected, the following
```

Comment:
This we need to think how to term this -  this does avoid the need to bundle all paid regions up with the other paid regions as soon as the rest of the world checkbox is checked - which makes me think this is a great place to start.

Should this mention USD?

Here's a suggestion for an amendment:

'As "Rest of World" is selected your app will be available for purchase by credit-card in all enabled "other regions" using USD"

Other regions could link to the Other Regions heading via a fragment identifer.